modules/virt-create-node-network-config-console.adoc

```diff
@@ -2,33 +2,30 @@
 [id="virt-create-node-network-config-console_{context}"]
 = Creating a policy
 
-You can create a policy by using either a form or YAML in the web console.
+You can create a policy by using either a form or YAML in the web console. When creating a policy using a form, you can see how the new policy changes the topology of the nodes in your cluster in real time.
 
 .Procedure
-. Navigate to *Networking* → *NodeNetworkConfigurationPolicy*.
+. Navigate to *Networking* → *Node Network Configuration*.
 
-. In the *NodeNetworkConfigurationPolicy* page, click *Create*, and select *From Form* option.
-+
-In case there are no existing policies, you can alternatively click *Create NodeNetworkConfigurationPolicy* to createa policy using form.
+. In the *Node Network Configuration* page, click *Create*, and select the *From Form* option.
 +
 [NOTE]
 ====
-To create policy using YAML, click *Create*, and select *With YAML* option. The following steps are applicable to create a policy only by using form.
+To create policy using YAML, click *Create*, and select the *With YAML* option. The following steps are applicable to create a policy only by using form.
 ====
-
 . Optional: Check the *Apply this NodeNetworkConfigurationPolicy only to specific subsets of nodes using the node selector* checkbox to specify the nodes where the policy must be applied.
 
 . Enter the policy name in the *Policy name* field.
-
 . Optional: Enter the description of the policy in the *Description* field.
+. Click *Next* to move to the *Policy Interfaces* section.
 
-. Optional: In the *Policy Interface(s)* section, a bridge interface is added by default with preset values in editable fields. Edit the values by executing the following steps:
+. In the *Bridging* part of the *Policy Interfaces* section, a bridge interface named _br0_ is added by default with preset values in editable fields. If required, edit the values by doing the following:
 
 .. Enter the name of the interface in *Interface name* field.
 
-.. Select the network state from *Network state* dropdown. The default selected value is *Up*.
+.. Select the required network state in the *Network state* part of the section. The default selected state is *Up*.
```

Comment:
```suggestion
.. Select the required network state in the *Network state* list. The default selected state is *Up*.
```
If it's what we would normally call a "dropdown menu," we use "list"

Reply:
@ousleyp This is not a dropdown menu and not a list. It's just several radio buttons, and each one selects a state. Maybe just "Select the required network state." would be enough.